tools/codegen/codespec/merge_attributes.go

```diff
@@ -0,0 +1,79 @@
+package codespec
+
+import "sort"
+
+func mergeAttributes(pathParams, createRequest, createResponse, readResponse Attributes) Attributes {
```

Comment:
This is super powerful! Having all the attributes in one place and merging. I like it 👍